gcloud-java-examples/pom.xml

```diff
@@ -22,6 +22,12 @@
       <artifactId>gcloud-java</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
```

Comment:
Agreed on using just shade. Though I discovered that even just putting the shade plugin in the pom I still end up with a source and documentation jar. I have no idea where those are coming from.